### PR TITLE
Add dict/json import/export to Quantities, Units and Entities

### PR DESCRIPTION
**Pull request by ad-hardy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,6 +116,37 @@ dimensionality:
...
+
+Similar export syntax applies to exporting Unit and Entity objects.
+
+You can import Entity, Unit and Quantity objects from dictionaries and JSON. This requires that the object was exported with `include_unit_dict=True` and `include_entity_dict=True` (as appropriate)
+
+```pycon
+>>> quant_dict = quant[0].to_dict(include_unit_dict=True, include_entity_dict=True)
+>>> quant = Quantity.from_dict(quant_dict)
+>>> ent_dict = Entity.to_json()
+>>> ent = Entity.from_json()
```
Comment (nielstron) [inline README.md:147]:
```suggestion
>>> ent = Entity.from_json(ent_dict)
```

File: README.md [hunk 2]
```diff
@@ -116,6 +116,37 @@ dimensionality:
...
+```
+
+Similar export syntax applies to exporting Unit and Entity objects.
+
+You can import Entity, Unit and Quantity objects from dictionaries and JSON. This requires that the object was exported with `include_unit_dict=True` and `include_entity_dict=True` (as appropriate)
+
+```pycon
+>>> quant_dict = quant[0].to_dict(include_unit_dict=True, include_entity_dict=True)
+>>> quant = Quantity.from_dict(quant_dict)
+>>> ent_dict = Entity.to_json()
```
Comment (nielstron) [inline README.md:146]: I think this does not make too much sense? It should be a concrete entity right?

**Review by nielstron — CHANGES_REQUESTED** [review]
LGTM, just some minor requests for the documentation

File: README.md (hunk 2 above)
Comment (ad-hardy) [reply README.md:146]: Sorry @nielstron, I'm not sure what you mean, could you try re-wording it?

Comment (nielstron) [reply README.md:146]: Does `Entity.to_json()` work? Should it not be an instance and not the class being called (I.e. `Entity(...).to_json()`?

Comment (ad-hardy) [reply README.md]: A yes I see, got myself a bit confused while writing that. Have comitted an update.

**Pull request merged**